ts/screens/profile/PrivacyMainScreen.tsx

```diff
@@ -273,20 +284,23 @@ const PrivacyMainScreen = ({ navigation }: Props) => {
         loadingOpacity={0.9}
         loadingCaption={I18n.t("profile.main.privacy.loading")}
       >
-        <FlatList
-          scrollEnabled={false}
+        <SectionList
+          sections={[
+            {
+              data: privacyNavListItems
+            }
+          ]}
           keyExtractor={(item: PrivacyNavListItem, index: number) =>
             `${item.value}-${index}`
           }
+          renderItem={renderPrivacyNavItem}
+          ItemSeparatorComponent={Divider}
           contentContainerStyle={{
             paddingHorizontal: IOVisualCostants.appMarginDefault
           }}
```

Comment:
Since `SectionList` extends a `React.PureComponent` I suggest memoizing properties where shallow comparison is ineffective. What I mean is that `sections`, `keyExtractor` and `contentContainerStyle` properties hold references to memory addresses that change with every render. Memoizing these properties can prevent unnecessary re-renders of `SectionList` and its content. What do you think?

Reply:
done! Thanks for this suggestion!